packages/twilio-worker-auth/src/addAccountSidMiddleware.ts

```diff
@@ -16,11 +16,13 @@
 
 import './index';
 import type { Request, Response, NextFunction } from 'express';
+import { AccountSID } from '@tech-matters/types';
 
 declare global {
   namespace Express {
     export interface Request {
-      accountSid?: string;
+      accountSid?: AccountSID;
+      hrmAccountId?: string;
```

Comment:
Since `accountSid` will now only be used as part of the auth, maybe we could nest it one lever deeper in the `req` object? Just to avoid confusing the usage of `accountSid` vs `hrmAccountId`? That or please be very observant on my next PR, as I'll probably use `accountSid` out of habit :laughing:

Reply:
I think beyond the auth, we use it during the contact permissions check and the permissions endpoint implementations . I was going to move it under `twilioUser` - I just didn't want to stink up this PR with loads of rename changes, will do that in a follow on (but thanks for the reminder).